README.md

```diff
@@ -116,6 +116,13 @@ rake db:create
 will create the above database when the `DB` environment variable is set to postgres or mysql.
 You should run this before running rake in order to ensure that the `cc_test` database exists.
 
+#### Installing mysql2
```

Comment:
While I understand the reasoning for this addition, I'm not sure if the README is the right place for such a specific workaround. It comes with some assumptions (e.g. that openssl has been installed with brew) and specifies a specific version (e.g. mysql2 v0.5.3) which can get easily out of sync with the Gemfile.